internal/webhook/external/helpers.go

```diff
@@ -0,0 +1,79 @@
+package external
```

Comment:
I know "helper" is a word I used at some point when suggesting this logic could be extracted from a platform-specific handler and made reusable by all the handlers, but I don't love `helpers.go` as a file name.

Everything in this file is specific to refreshing something. (Except for `GenerateWebhookPath()`, which can probably move to somewhere like `path.go`.) I think we'd be better off naming this file `refresh.go` and then it's more obvious from a glance at the dir tree what must be in here.